shopinvader_sale_communication/services/abstract_sale.py

```diff
@@ -12,8 +11,12 @@ def _convert_one_sale(self, sale):
         values = super(AbstractSaleService, self)._convert_one_sale(sale)
         values.update(
             {
-                "online_information_for_customer": sale.online_information_for_customer,
-                "online_information_request": sale.online_information_request,
+                "online_information_for_customer": sale.online_information_for_customer
+                if sale.online_information_for_customer != "<p><br></p>"
+                else False,
```

Comment:
```suggestion
                else None,
```
False is for Boolean ;)

Reply:
My bad! Thanks for spotting it :))